include/drivers/usbc/usbc_tcpc.h

```diff
@@ -0,0 +1,1020 @@
+/* usbc_tcpc.h - USB Type-C Port Controller driver interface */
+
+/*
+ * Copyright 2022 The Chromium OS Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief USBC Type-C Port Controller device APIs
+ *
+ * This file contains the USB Type-C Port Controller device APIs.
+ * All Type-C Port Controller device drivers should implement the
+ * APIs described in this file.
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_USBC_USBC_TCPC_H_
+#define ZEPHYR_INCLUDE_DRIVERS_USBC_USBC_TCPC_H_
+
+/**
+ * @brief USB Type-C Port Controller API
+ * @defgroup _usb_type_c_port_controller_api USB Type-C Port Controller API
+ * @{
+ */
```

Comment:
This defines a top group, however, all drivers now go into `io_interfaces` (check others)